### ISSUE TITLE
Sending any array to withHeaders() removes webhook-server config headers

### ISSUE THREAD

**Issue by The-l3asics** [spatie/laravel-webhook-server#78]
Version: **1.11.2**
Laravel Version: **8.35.1**
Php Version: **7.4.1**
Issue: When adding headers to a webhook, all predefined headers are removed from the request.
Config file like the following:
```php
<?php
 /*
     * These are the headers that will be added to all webhook requests.
     */
    'headers' => [
        'Content-Type' => 'application/json',
    ],
```
And the webhook call that I am using:
```php
<?php
WebhookCall::create()
            ->url($webhookUrl)
            ->meta($meta)
            ->withHeaders($headers)
            ->payload($payload)
            ->doNotSign()
            ->dispatch();
```
Regardless of the value that headers is, the default headers are removed. I have tried actual headers as well an empty ([]) array but it still seems to remove the predefined ones. Any new headers **do** make it into the request.

An example of not using withHeaders():
![image](https://user-images.githubusercontent.com/7373119/113389928-d5510e00-935e-11eb-9f2c-4ee7c3bca0d6.png)
Example of using withHeaders($headers):
![image](https://user-images.githubusercontent.com/7373119/113389992-eef25580-935e-11eb-85f1-461eba84230e.png)

**Comment by freekmurze** [comment]
I accept a PR with tests that fixes this.

**Comment by wotta** [comment]
Is there a timeframe when this issue should be solved?
I have some time to fix this tonight so not sure if that's okay 🙂

**Comment by freekmurze** [comment]
There are no deadlines in open source 👍

**Comment by wotta** [comment]
Nice, Just making sure. I did look into the code and think that I see what's going wrong.

**Comment by freekmurze** [comment]
Fixed by #79
